ci/jenkins/test-vmc.sh

```diff
@@ -371,7 +375,7 @@ function deliver_antrea {
     for i in "${!IPs[@]}"
     do
         ssh-keygen -f "/var/lib/jenkins/.ssh/known_hosts" -R ${IPs[$i]}
-        copy_image antrea-ubuntu.tar projects.registry.vmware.com/antrea/antrea-ubuntu ${IPs[$i]} latest true
+        copy_image antrea-ubuntu.tar docker.io/antrea/antrea-ubuntu ${IPs[$i]} latest true
```

Comment:
Is `docker.io/` required? I thought `antrea/antrea-ubuntu` automatically resolve to `docker.io/antrea/antrea-ubuntu`.

Reply:
for `ctr` seems not, i remeber i have to change this because faced some error when not adding this.

Reply:
agree with Hang. I also once hit into error if not adding `docker.io/`